test/helpers/choice_helper.rb

```diff
@@ -28,14 +28,15 @@ def create_full_choice(attribs={}, alt_ct=2)
   end
 
   def choice_params(choice)
+    choice.populate_dates_and_times
```

Comment:
a little trick that enables setting the column values in the test, and translating to split values for the params. It uses the model method to populate the split value strings.